pkg/systemd/systemd.go

```diff
@@ -0,0 +1,28 @@
+package systemd
+
+import (
+	"fmt"
+	"os/exec"
+	"strconv"
+	"strings"
+)
+
+// Version retrieves the systemd major version by executing the "systemctl --version" command and parsing its output.
+// It returns the parsed version as an uint64, or an error if the command execution or parsing fails.
+func Version() (uint64, error) {
+	cmd := exec.Command("systemctl", "--version")
```

Comment:
Feel like we should rather check for udev rather than systemd:

```suggestion
	cmd := exec.Command("udevadm", "--version")
```

Reply:
There ways to get udev without systemd.

Reply:
Not sure what mudev actually returns here.

Reply:
Made the change to `udevadm`.